examples/projections/azim/azim_equidistant.py

```diff
@@ -8,11 +8,13 @@
 that lie within a certain distance or for comparing distances of different
 locations relative to the projection center.
 
-``elon0/lat0[/horizon]/scale`` or ``Elon0/lat0[/horizon]/width``
+**e**\ *lon0*\ **/**\ *lat0*\ [**/**\ *horizon*]\ **/**\ *scale* or
+**E**\ *lon0*\ **/**\ *lat0*\ [**/**\ *horizon*]\ **/**\ *width*
```

Comment:
@willschlitzer No need to make it too complicated.

This is what GMT does:
```
**-Je**\ *lon0/lat0*\ [*/horizon*]\ */scale*
```

Reply:
Okay, I was treating the forward slashes as literal arguments. The GMT version looks much cleaner!